lizard_auth_client/models.py

```diff
@@ -153,3 +153,48 @@ def get_organisation_with_role(user, rolecode):
     Organisation.MultipleObjectsReturned if he has more than one.
     """
     return get_organisations_with_role(user, rolecode).get()
+
+
+def get_user_org_role_dict(user):
+    """
+    :returns a dict for a given user containing information about the user
+    itself, the organisations he/she belongs to, and the specific permissions
+    (or roles) he/she has within these organisations::
+
+        {u'email': u'<str>',
+         u'is_superuser': <bool>,
+         u'username': u'<str>',
+         u'organisations': [{u'id': u'<str>',
+                             u'name': u'<str>',
+                             u'permissions': [u'<str>', u'<str>']},
+                            {u'id': u'<str>',
+                             u'name': u'<str>.',
+                             u'permissions': [u'<str>']}],
+         }
+
+    :returns None if there is no relation  between the user and an organisation.
```

Comment:
Mee eens, liever dat als er geen enkele user-organisation-role relatie is, dat dan ``organisations`` een lege lijst is.